react/features/base/react/components/native/NavigateSectionListItem.js

```diff
@@ -129,37 +138,59 @@ export default class NavigateSectionListItem extends Component<Props> {
      * @returns {ReactElement}
      */
     render() {
-        const { colorBase, lines, title } = this.props.item;
+        const { slideActions } = this.props;
+        const { id, colorBase, lines, title } = this.props.item;
         const avatarStyles = {
             ...styles.avatar,
             ...this._getAvatarColor(colorBase)
         };
+        let right;
+
+        // NOTE: The {@code Swipeout} component has an onPress prop encapsulated
+        // in the {@code right} array, but we need to bind it to the ID of the
+        // item too.
+
+        if (slideActions) {
+            right = [];
+            for (const slideAction of slideActions) {
+                right.push({
+                    backgroundColor: slideAction.backgroundColor,
+                    onPress: slideAction.onPress.bind(undefined, id),
+                    text: slideAction.text
+                });
+            }
+        }
 
         return (
-            <Container
-                onClick = { this.props.onPress }
-                style = { styles.listItem }
-                underlayColor = { UNDERLAY_COLOR }>
-                <Container style = { styles.avatarContainer }>
-                    <Container style = { avatarStyles }>
-                        <Text style = { styles.avatarContent }>
-                            {title.substr(0, 1).toUpperCase()}
+            <Swipeout
+                backgroundColor = { ColorPalette.transparent }
+                right = { right }>
+                <Container
+                    onClick = { this.props.onPress }
+                    style = { styles.listItem }
+                    underlayColor = { UNDERLAY_COLOR }>
+                    <Container style = { styles.avatarContainer }>
+                        <Container style = { avatarStyles }>
+                            <Text style = { styles.avatarContent }>
+                                {title.substr(0, 1).toUpperCase()}
+                            </Text>
+                        </Container>
+                    </Container>
+                    <Container style = { styles.listItemDetails }>
+                        <Text
+                            numberOfLines = { 1 }
+                            style = {{
+                                ...styles.listItemText,
```

Comment:
why use destructuring instead of an array of styles?

Reply:
Good question, why? 😆 

Fixed, thanks!